MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Selection in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Selection in Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
```

Comment:
Learn about the segment item selection in Syncfusion .NET MAUI Segmented Control (SfSegmentedControl).

Reply:
changed description